code/style.css

```diff
@@ -1,7 +1,205 @@
+    right: 0.3rem;
+    top: 11rem;
+    font-size: 1rem;
+    color: white;
+    background-color: black;
+    opacity: 0.5;
+}
+
+h1 {
+    font-size: 1.3rem;
+    margin-bottom: 0.5rem;
+    text-align: center;
+}
+
+h2 {
+    font-size: 1rem;
+    margin-bottom: 0.5rem;
+}
+
+.main {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-gap: 1rem;
+    padding: 0.75rem;
+}
+
+.signup {
+    width: 100%;
+    background-color: grey;
+    display: flex;
+    padding: 0.5rem;
+    flex-direction: column;
+    max-width: 500px;
+    margin-left: auto;
+    margin-right: auto;
+    border: 2px solid darkgray;
+    border-radius: 10px;
+    overflow: hidden;
+}
+
+.preference-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+}
+
+.checkbox {
+    display: inline-flex;
+    cursor: pointer;
+    position: relative;
+}
+
+    .checkbox > span {
+        color: black;
+        padding: 0.5rem 0.25rem;
+    }
+
+    .checkbox > input {
+        height: 25px;
+        width: 25px;
+        -webkit-appearance: none;
+        -moz-appearance: none;
+        border: 1px solid #34495E;
+        border-radius: 4px;
+        outline: none;
+        transition-duration: 0.3s;
+        background-color: white;
+        cursor: pointer;
+    }
+
+        .checkbox > input:checked {
+            border: 1px solid rgb(10, 88, 88);
+            background-color: lightgray;
+        }
+
+            .checkbox > input:checked + span::before {
+                content: '\2713';
+                display: block;
+                text-align: center;
+                color: rgb(10, 88, 88);
+                position: absolute;
+                left: 0.7rem;
+                top: 0.2rem;
+            }
+
+        .checkbox > input:active {
+            border: 2px solid #34495E;
+        }
+
+.form-field {
+    display: block;
+    margin: 0.5rem 0;
+}
+
+.faq-container {
+    width: 100%;
+    background-color: lightgray;
+    padding: 0.5rem;
+    border: 2px solid darkgray;
+    display: flex;
+    flex-direction: column;
+    border-radius: 10px;
+    overflow: hidden;
+}
+
+.submit-button {
+    width: 10rem;
+    height: 2rem;
+    background-color: rgb(10, 88, 88);
+    color: white;
+}
+
+.accordion-button {
+    display: block;
+    width: 100%;
+    min-height: 4rem;
+    padding: 1rem;
+    border: none;
+    outline: none;
+    cursor: pointer;
+    background-color: rgb(10, 88, 88);
+    color: white;
+    text-align: left;
+    transition: background-color 1s;
+    margin-bottom: 0.5rem;
+}
+    /* Unicode for arrow down 25BC*/
+    .accordion-button::after {
+        content: '\25bc';
+        float: right;
+    }
+
+.accordion-button--active {
+    background: rgb(157, 206, 66)
+}
+
+    .accordion-button--active::after {
+        content: '\25b2'
+    }
+
+.accordion-content {
+    padding: 0 15px;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    background: #eeee;
+    display: none;
+    overflow: auto;
```

Comment:
Thumbs up for the accordion, I'm impressed!
The change of the background color and you have also added an icon that indicates
whether the section is expanded or collapsed.